package.json

```diff
@@ -20,7 +20,10 @@
     "generate:meetings": "node scripts/build-meetings.js",
     "generate:dashboard": "node scripts/dashboard/build-dashboard.js",
     "generate:videos": "node scripts/build-newsroom-videos.js",
-    "generate:tools": "node scripts/build-tools.js"
+    "generate:tools": "node scripts/build-tools.js",
+    "storybook": "storybook dev -p 6006",
+    "chromatic": "chromatic --exit-zero-on-changes",
+    "build-storybook": "storybook build"
   },
```

Comment:
Make the script commands according to the pattern followed, like `build:storybook`. Follow the same for all the scripts added.

Reply:
Done